dongkyu/initial-settings-express/myapp/app.js

```diff
@@ -1,13 +1,12 @@
-
 const express = require("express");
 const cors = require("cors");
 const morgan = require("morgan");
 const dotenv = require("dotenv");
 dotenv.config();
 
-const { DataSource } = require("typeorm");
+const { DataSource, Column } = require("typeorm");
```

Comment:
```suggestion
const { DataSource } = require("typeorm");
```
필요하지 않으므로 지워주셔도 될 것 같습니다-!